src/main/webapp/app/overview/course-lectures/course-lecture-details.component.scss

```diff
@@ -0,0 +1,9 @@
+.wrapper-btn-manage-lecture {
+    display: flex;
+    align-items: center;
+}
+
+.btn-manage-lecture {
+    height: fit-content;
+    background-color: var(--secondary) !important;
```

Comment:
You can use the CSS class `btn-secondary` instead of forcing the secondary background color on the default button. Also, if you decide to move the button down so that it is placed like for exercises, a different color might even better.

Reply:
Done